indico/modules/events/registration/templates/management/_badge_template.html

```diff
@@ -0,0 +1,9 @@
+<ul class="badge-template-list">
+    {% for template_id, template in templates %}
+        <li>
+            <input type="radio" name="templateId" value="{{ template_id }}" id="{{ template_id }}"
+                   checked="{% if loop.first %}checked{% endif %}">
```

Comment:
That is wrong. The sole existence of `checked` makes it checked. No matter if it's empty or 0 or false or whatever else.

You want `{% if loop.first %}checked{% endif %}` to create the attribute only in the first loop